script/verify-snapshot-script

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
```

Comment:
Isn't naming this file verify-snapshot-*script* a little redundant when it already is in a folder named script?

Reply:
"Script" in this case refers to the snapshot script that gets generated by electron-link, because we want to verify it can be run via `mksnapshot`. Omitting it would be misleading, as `verify-snapshot` seems to indicate we're about the verify the actual snapshot blob, as opposed to the script it is generated from.